tests/test.yml

```diff
@@ -3,6 +3,20 @@
 
   vars:
     nginx_use_ppa: true
+    nginx_remove_default_vhost: true
+    nginx_vhosts:
+      - server_name: "test.dev"
+        root: "/var/www/test"
+
+  pre_tasks:
+    - name: Update apt cache.
+      apt: update_cache=yes cache_valid_time=86400
+      when: ansible_os_family == 'Debian'
+
+    - name: Install dependencies.
+      package: name={{ item }}
```

Comment:
If it's just one, we can set `package: name=curl`. Leave it nice and tidy until we need `items`.